src/main.rs

```diff
@@ -11,7 +11,7 @@ use std::fs::File;
 use std::io::{self, BufWriter, Write};
 
 use clap::{Arg, App};
-use ring::rand::SystemRandom;
+use ring::rand::{SecureRandom, SystemRandom};
```

Comment:
This is required because now `fill()` is only on the `SecureRandom` trait, not directly on `SystemRandom` anymore.